meson.build

```diff
@@ -221,10 +222,13 @@ config_h = configure_file(
 setcap = find_program('setcap', '/usr/sbin/setcap', '/sbin/setcap', required : false)
 if get_option('NO_SETCAP_OR_SUID')
 	perm_type = 'none'
+	setcap_path = '/dev/null'
```

Comment:
BTW I'm not sure if that's the best idea, but I didn't have a better one and it should at least result in a failure if the script would ever use it:
```bash
$ /dev/null
-bash: /dev/null: Permission denied
```

And `build-aux/setcap-setuid.sh` only requires `setcap_path` if `perm_type == 'caps'`.
Alternatively we could e.g. use a path that doesn't exist.